module1.js

```diff
@@ -29,6 +31,11 @@ function question1(name1) {
 
 function question2(number1, number2) {
   // TODO:
+  if (number1 == number2) {
```

Comment:
ditto, although it works but no reason to use `==`, `===` is best practice as it is a more strict check and could be condensed to one line, can you think how?